hack/ci/install-kind.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#     Copyright 2020 Nexus Operator and/or its authors
+#
+#     This file is part of Nexus Operator.
+#
+#     Nexus Operator is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     Nexus Operator is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with Nexus Operator.  If not, see <https://www.gnu.org/licenses/>.
+
+set -e
+default_kind_version="v0.8.1"
+
+if [[ -z ${KIND_VERSION+x} ]]; then
```

Comment:
Nice, I didn't know about [this](https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash/13864829#13864829).